README.md

```diff
@@ -8,61 +8,41 @@ Place the following HTML where you’d like the component to appear (this works
 or even no framework!):
 
 ```html
-<manifold-plan-matrix></manifold-plan-matrix>
+<manifold-init client-id="[your client ID]"></manifold-init>
+<manifold-plan-table></manifold-plan-table>
 ```
 
-This component also requires our [manifold-init][manifold-init] component to be present on the page
-with your client ID added.
-
-```html
-<manifold-init client-id="your client ID"></manifold-init>
-```
```

Comment:
Because of analytics and metrics reasons the cleint-id is required on both components currently. Perhaps the example should continue to reflect that?

Reply:
Yeah good call. Hopefully we can fix that quickly, but you’re right when that’s changed we’ll update the README

Reply:
thanks! I will approve. Ya if we had common analytics tooling in manifold-init or something else that would solve the problem - not sure if mui is tracking it. The current Track function is not suffiecient as its not to parity with our current analytics set up.